components/searchable_user_list/searchable_user_list.jsx

```diff
@@ -84,14 +82,14 @@ class SearchableUserList extends React.PureComponent {
         this.nextTimeoutId = setTimeout(() => this.setState({nextDisabled: false}), NEXT_BUTTON_TIMEOUT);
 
         this.props.nextPage();
-        $(ReactDOM.findDOMNode(this.refs.channelListScroll)).scrollTop(0);
+        this.refs.userList.scrollToTop();
```

Comment:
Funny that no one ever noticed this didn't work before. I guess jQuery caused that to fail silently

Reply:
Yes, I guess so.